tests/test_integration.py

```diff
@@ -92,6 +93,7 @@ class DebugToolbarIntegrationTestCase(TestCase):
     def test_middleware(self):
         response = self.client.get("/execute_sql/")
         self.assertEqual(response.status_code, 200)
+        self.assertContains(response, "djDebug")
```

Comment:
This caused the test to fail until the views returned a response containing `</body>`.